### PR TITLE
Use ScrollView in examples a manual_tests

### PR DESCRIPTION
**Pull request by abarth**
This patch deploys ScrollView in a number of demos and manual tests.

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie

**Comment by Hixie** [comment]
LGTM.

Summary of our discussion around this:
- we won't include the shrine demo yet
- there are three bugs to look into:
  - why does it go past the max scroll offset?
  - can we fix the numeric issue with the android simulation whereby it's not idempotent (if you create it with its current velocity, you get a different velocity)
  - can we figure out some magic way to avoid the missed frame when recreating the ballistic simulation?

**Comment by Hixie** [comment]
This patch substantially improved the performance of the microbenchmarks.stock_animation_subsequent_frame and microbenchmarks.stock_animation_close_first_frame benchmarks.

**Comment by Hixie** [comment]
It also increased code size for complex_layout aot_snapshot_size_instructions, but I expect that to reduce when we delete the old code which I presume is still somehow getting linked in here.

File: examples/stocks/lib/stock_home.dart [hunk 1]
```diff
@@ -121,63 +121,65 @@ class StockHomeState extends State<StockHome> {
...
-        new DrawerItem(
-          icon: new Icon(Icons.settings),
-          onPressed: _handleShowSettings,
-          child: new Text('Settings')),
-        new DrawerItem(
-          icon: new Icon(Icons.help),
-          onPressed: _handleShowAbout,
-          child: new Text('About'))
-      ])
+      child: new ScrollView(
```
Comment (Hixie) [inline examples/stocks/lib/stock_home.dart]: This change regressed the benchmark I just made from 3621.0 to 4519.3.

**Pull request merged**